src/index.jsx

```diff
@@ -49,6 +50,7 @@ ReactDOM.render(
       <Route path='/' component={Layout}>
         <IndexRoute name='home' component={Home} onEnter={boundRouteActions('home')}/>
         <Route path='about' component={About} />
+        <Route path='dev' component={Dev} />
```

Comment:
If we want to use `/dev`, we should add it to the username blacklist in `freefeed-server`.